images/instrumentation/injector/src/auxv.zig

```diff
@@ -8,7 +8,7 @@ const print = @import("print.zig");
 // Executables requiring getauxval will bind to libc's getauxval, not the symbol exported here.
 pub export fn getauxval(auxv_type: u32) callconv(.C) usize {
     var auxv_file = std.fs.openFileAbsolute("/proc/self/auxv", .{}) catch |err| {
-        print.printMessage("Failed to open /proc/self/auxv: {}", .{err});
```

Comment:
The injector previously did not differentiate between log levels besides debug, info and none, so every error message was printed basically at level info. Now, with more log levels available, every error message changed from printMessage to printError, which makes the diff a bit noisy.